lib/rules/prefer-destructuring.js

```diff
@@ -82,26 +82,18 @@ module.exports = {
     },
     create(context) {
 
-        let checkArrays = true;
-        let checkObjects = true;
-        let enforceForRenamedProperties = false;
         const enabledTypes = context.options[0];
         const additionalOptions = context.options[1];
+        let enforceForRenamedProperties = false;
```

Comment:
Nitpick: I think this can just be `let enforceForRenamedProperties = context.options[1] && context.options[1].enforceForRenamedProperties;`